hidden/Features/StatusBar/MouseMovementHandler.swift

```diff
@@ -52,12 +52,10 @@ class MouseMovementHandler {
         }
         var menubarMinY: CGFloat = .nan
         
-        if let menubarWindow = WindowInfo.windowNamed("Menubar") {
-            menubarMinY = screenHeight - menubarWindow.frame.height
-        }
-        
-        if menubarMinY == .nan {
-            print("Unable to get Menubar window height")
+        if let menubarFrame = WindowInfo.cgRectOfWindow(named: "Menubar") {
+            menubarMinY = screenHeight - menubarFrame.height
+        } else {
+            print("Failed to setup MouseMovementEventMonitor: Unable to get Menubar window height")
             return
         }
         
```

Comment:
The mouse inverted because Cocoa using top left coordinator system but CoreGraphic using bottom left, you need convert between them, I'm so confused at first time like you.